tests/commands/test_commit_command.py

```diff
@@ -52,6 +52,26 @@ def test_commit_retry_fails_no_backup(config, mocker):
     assert NoCommitBackupError.message in str(excinfo.value)
 
 
+@pytest.mark.usefixtures("staging_is_clean")
+def test_commit_allow_empty(config, mocker):
+    prompt_mock = mocker.patch("questionary.prompt")
```

Comment:
Overall the code looks good to me. Could you please explain a bit more on this test case? I'm a bit confused

Reply:
Yep, this test does not quite make sense. As what we're trying is to make an empty commit, passing "prefix" and "subject" might not be what we want.